### DISCUSSION TITLE
How do I know which container of this path belongs to

### DISCUSSION THREAD

**Discussion by jixawang** [k3s-io/k3s#10406] [Q&A]
**Environmental Info:**
K3s Version: 
k3s version v1.23.16+k3s1 (64b0feeb)
go version go1.19.5

Node(s) CPU architecture, OS, and Version: 
Linux xxx 3.10.0-1160.118.1.el7.x86_64 #1 SMP Wed Apr 24 16:01:50 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux

Cluster Configuration:
3 servers

**Describe the bug:**
Plugin Output: 
  Path             : /var/lib/rancher/k3s/agent/containerd/io.containerd.snapshotter.v1.overlayfs/snapshots/232/fs/usr/lib/x86_64-linux-gnu/libssl.so.1.0.0
  Reported version : 1.0.1d
  Fixed version    : 1.0.1q

Hello Dears, I need to fix a cyber security issues, How can I determine the corresponding container(or image) for this path?

Thank you so much.

![图片](https://github.com/k3s-io/k3s/assets/34025537/5adee0cd-f0d4-43a5-b99a-35670f06c0dc)

**Comment by brandond** [comment]
I would probably ask at https://github.com/containerd/containerd - this is a question about containerd's overlayfs snapshotter, not k3s.
